internal/envelope/cid.go

```diff
@@ -0,0 +1,49 @@
+package envelope
+
+import (
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime"
+	"github.com/ipld/go-ipld-prime/codec/dagcbor"
+	"github.com/libp2p/go-libp2p/core/crypto"
+	"github.com/multiformats/go-multibase"
+	"github.com/multiformats/go-multicodec"
+	"github.com/multiformats/go-multihash"
+)
+
+var b58BTCEnc = multibase.MustNewEncoder(multibase.Base58BTC)
+
+// CIDToBase56BTC is a utility method to convert a CIDv1 to the canonical
+// string representation used by UCAN.
+func CIDToBase58BTC(id cid.Cid) string {
+	return id.Encode(b58BTCEnc)
+}
+
+// CID returns the UCAN content identifier a Tokener.
```

Comment:
Those functions that do encoding and/or signature should be clearly marked as expensive.
This one in particular is only used in tests, maybe that should be private?

Reply:
That one is internal and can't be used outside `go-ucan`